### ISSUE TITLE
[Core] Healthstone importance

### ISSUE THREAD

**Issue by joshinat0r** [WoWAnalyzer/WoWAnalyzer#2289]
The Healthstone is right now always a minor suggestion. It might be a good idea to bumb up it's importance when the player died or when the difficulty is heroic / mythic.

Not about the level, might be a good idea to make it a major issue when the player didn't used it and died and an average issue when the difficulty is heroic / mythic.
Feel free to suggest a different importance.

**Comment by MartijnHols** [comment]
I think it should always be minor in heroic or mythic if the player hasn't died. Saving it for when you might die pretty much always beats using it to relieve healers.

As for dying, we already have the "you died" suggestion. I don't think we should have two suggestions for that, so instead I suggest mentioning it explicitly in the "you died" suggestion instead - if that's not already the case.

**Comment by viridis** [comment]
Is there a way to check for health dips to like <30% health or something where the incoming heals are not 'ready' to pick up the person from that? Using a healthstone - even in heroic/mythic - where it is not urgent doesn't feel like a super good suggestion (Ghuun laser phases where periodically everyone drops to 65-70% hp but healers are (or should be) prepared for it, and thus a healthstone will be kind of a waste anyway.

**Comment by joshinat0r** [comment]
The death-suggestion already mentions healthpots/stones.
![image](https://user-images.githubusercontent.com/29842841/46204700-7427ca00-c31e-11e8-9706-c96a4e4361b3.png)

As for checking health dips or something like this, not a huge fan of it. Eg. a BDK on Fetid is tanking the thrash with his EHP and DSs right after the hit and they need to be full HP to do so. Which means usually they're saving healing pots in case healers couldn't top him off in time (and DS isn't going to heal for much since the 5sec window is over). Long story short, he will dip low quite frequently and we would make it a major issue when he save it in reality as backup to top himself off.
Quite a specific example but I'm fairly sure that a module would have to account for such things or we have to add stuff manually (and we don't like stuff that needs continuously adjustments).

I think simply expanding the death-suggestion to mention something like `Your health stone wasn't on cooldown when you did` should be enough (and is fairly easy to add). It wouldn't add a new suggestion and it wouldn't be an over engineered module.

**Comment by MartijnHols** [comment]
Fixed in b4ff1b1292ad27bd040a48e0b315a7549d142f48

I added healthstone and healing potions to the death recap, and reworded the suggestions to point users to that. This should give them all the info they need.

I removed the extra suggestions for healthstones and healing potions to reduce clutter; they're only *necessary* when to prevent deaths, so complaining in any other case does not help the user and in fact only detracts from other suggestions.
